hack/Dockerfile

```diff
@@ -56,8 +51,48 @@ RUN set -ex \
 	&& dpkg -i /tmp/repo-mediaarea_1.0-21_all.deb \
 	&& apt-get update \
 	&& apt-get install -y --no-install-recommends \
-		libyaml-dev clamav nodejs \
-	&& /src/hack/osdeps.py Ubuntu-22 1 | grep -v -E "nginx|postfix|python3.9-dev" | xargs apt-get install -y --no-install-recommends \
+		atool \
+		bulk-extractor \
+		coreutils \
+		ffmpeg \
+		fits \
+		g++ \
+		gcc \
+		gearman \
+		gettext \
+		ghostscript \
+		hashdeep \
+		imagemagick \
+		inkscape \
+		jhove \
+		libffi-dev \
+		libimage-exiftool-perl \
+		libldap2-dev \
+		libmysqlclient-dev \
+		libsasl2-dev \
+		libssl-dev \
+		libxml2-dev \
+		libxslt1-dev \
+		libyaml-dev \
+		logapp \
+		md5deep \
+		mediaconch \
+		mediainfo \
+		nailgun \
+		nfs-common \
+		openjdk-8-jre-headless \
+		p7zip-full \
+		pbzip2 \
+		pst-utils \
+		rsync \
+		siegfried \
+		sleuthkit \
+		tesseract-ocr \
+		tree \
+		unar \
+		unrar-free \
+		uuid \
+		clamav nodejs \
```

Comment:
Could we split each package separately and sort them? `clamav` and `nodejs` are two different packages.

There are also a few that are repeated from the top instruction like `gettext`, `libssl-dev`, etc. Should those be removed from this second instruction? Maybe combining both instructions makes sense.

Reply:
Thanks for the suggestions. Done